src/base/bittorrent/filesearcher.cpp

```diff
@@ -69,5 +73,6 @@ void FileSearcher::search(const BitTorrent::TorrentID &id, const PathList &origi
         findInDir(usedPath, adjustedFileNames, forceAppendExt);
     }
 
-    emit searchFinished(id, usedPath, adjustedFileNames);
+    promise.addResult(FileSearchResult {usedPath, adjustedFileNames});
```

Comment:
You can use designated initializers.
https://en.cppreference.com/w/cpp/language/aggregate_initialization#Designated_initializers

Reply:
> You can use designated initializers.

Ok.